### PR TITLE
Topic b more lines

### PR DESCRIPTION
**Pull request by AndreasDal**
two new lines added for clarity..

### CODE REVIEW

File: file1 [hunk 1]
```diff
@@ -2,4 +2,5 @@ Hello from Adal76 on 13/10
 Hello from AndreasDal on 13/10
 Hello again from Adal76 on 13/10
 Hello again from AndreasDal on 13/10
```
Comment (AndreasDal) [inline file1:4]: this line is really important!

Comment (Adal76) [reply file1:4]: Ok, I see what you mean! I will have a closer look at it and get back.

File: file1 [hunk 2]
```diff
@@ -1,3 +1,6 @@
 Hello from Adal76 on 13/10
 Hello from AndreasDal on 13/10
 Hello again from Adal76 on 13/10
+Hello again from AndreasDal on 13/10
+yet another one from AndreasDal on 13/10
+ups, forgot this line from AndreasDal on 14/10
```
Comment (Adal76) [inline file1]: I don't understand this line? is it really necessary to include it?

1.  extra work has to be done
2. risk of complicating the code ..

**Review by Adal76 — COMMENTED** [review]
test review comment: I await your argument before approving.

File: file1 (hunk 1 above)
Comment (Adal76) [reply file1:4]: ohh, I see that you added a new line since I commented last! I have a look at it.

File: file1 [hunk 3]
```diff
@@ -3,4 +3,5 @@ Hello from AndreasDal on 13/10
 Hello again from Adal76 on 13/10
 Hello again from AndreasDal on 13/10
 yet another one from AndreasDal on 13/10
-ups, forgot this line from AndreasDal on 14/10
\ No newline at end of file
+ups, forgot this line from AndreasDal on 14/10
+and one more line from AndreasDal on 14/10
```
Comment (Adal76) [inline file1:7]: Please remember to notify me when you add changes to the pull request!!

File: file1 (hunk 2 above)
Comment (Adal76) [reply file1]: 👍 I  👀 what you mean. but what do you mean by this:
>  this line is really important!

Comment (AndreasDal) [reply file1]: Just do it man, it is great stuff..

File: file1 (hunk 3 above)
Comment (AndreasDal) [reply file1:7]: ### got you ### 
yes, you're right, sorry I forgot 😸 

> lucky is the one who flies high in the sky

- [x] first task is to check for errors in code
- [x] second is to make it look nice
- [x] finally check if it functions as planned ..

**Comment by AndreasDal** [comment]
Hi collaborator!!! As you can see all my changes can be applied ....

File: file1 (hunk 1 above)
Comment (Adal76) [reply file1:4]: I see your point and rest my case ..

File: file1 (hunk 3 above)
Comment (Adal76) [reply file1:7]: I resolve this conversation and am ready to merge ..

**Comment by Adal76** [comment]
yes, you are right my friend. I will merge in the changes now ..

**Comment by Adal76** [comment]
### ready to merge ###
After a thorough review and conversations with collaborator I trust that everything is now fine to merge in 🍰 🎉

**Pull request merged**